ballerina/stream_types.bal

```diff
@@ -78,6 +78,70 @@ public class PersistSQLStream {
     }
 }
 
+
+public class PersistGoogleSheetsStream {
```

Comment:
```suggestion
public class GoogleSheetsStream {
```

Reply:
we don't need Persist prefix here

Reply:
We use the same convention for `PersistSQLStream`